jobs/scripts/upload_nupkg.sh

```diff
@@ -47,11 +47,11 @@ nuget_push() {
     fi
 }
 
-for nupkg in $( find ${nupkg_dir} -iname "*.nupkg" -not -iname "*symbols*" -not -iname "*transport.runtime.tizen.5.0.0-armel.Microsoft.NETCore.TestHost*" -not -iname "Microsoft.Extensions.DependencyModel*" ); do
+for nupkg in $( find ${nupkg_dir} -iname "*.nupkg" -not -iname "*symbols*" -not -iname "*transport.runtime.tizen.5.0.0-armel.Microsoft.NETCore.TestHost*" -not -iname "Microsoft.*" -not -iname "transport.Microsoft.*" ); do
```

Comment:
Why have you removed  `Microsoft.Extensions.DependencyModel*`?

Reply:
It was removed previously, because job was unable to push this nuget package (see http://52.79.132.74:8080/job/release/job/core-setup_master_armel_Release/683/console)

Reply:
Same for `*runtime.tizen.5.0.0-armel.Microsoft.NETCore.TestHost*` symbols packages